tests/utils.go

```diff
@@ -2519,7 +2519,7 @@ func WaitForConfigToBePropagatedToComponent(podLabel string, resourceVersion str
 		pods, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: podLabel})
 
 		if err != nil {
-			return fmt.Errorf("failed to fetch pods. %s", errComponentInfo)
+			return fmt.Errorf("failed to fetch pods: %s, %s", err.Error(), errComponentInfo)
```

Comment:
Any reason for not using the common `%v`?

Reply:
error type has a dedicated method `Error()` just for that, and I'm pro-standard methods.
But the end result is the same so I can change if you'd like.

Reply:
since both of you think so, I will change :) 
[Done](https://github.com/kubevirt/kubevirt/compare/d2265e34f113c8c0d7f78d37dcb47ee844d2e7af..117a4591518b84079b491d997c01edf16eddfabf)